app/Jobs/CreateCallPowerPostInRogue.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Chompy\Jobs;
+
+use Exception;
+use Chompy\Services\Rogue;
+use Illuminate\Http\Request;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+
+class CreateCallPowerPostInRogue implements ShouldQueue
+{
+	use Dispatchable, InteractsWithQueue, Queueable;
+
+	/**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+	public function __construct()
+	{
+	}
+
+    /**
+     * Execute the job to create a CallPower post in Rogue.
+     *
+     * @return array
+     */
+    public function handle(Rogue $rogue, Request $request)
+    {
+		// Using the mobile number, get or create a northstar_id.
```

Comment:
And here! ⬅️

Reply:
ah sorry about all of these @DFurnes ! This is so weird though - I reindented in my editor and it doesn't look like this there! Or am I missing something and need new 👀 ?
![Uploading Screen Shot 2019-03-22 at 11.47.29 AM.png…]()

Reply:
This is probably due to indentation settings being mismatched in your editor – see my comment below about copying over `.editorconfig` & double-check you have [the Sublime Text plugin](https://github.com/sindresorhus/editorconfig-sublime).

Reply:
Thanks @DFurnes ! I thought I had that but looks like I don't when I go to `Preferences>Browse Packages...`:
![Screen Shot 2019-03-22 at 12 28 42 PM](https://user-images.githubusercontent.com/9019452/54838021-15745500-4c9e-11e9-881d-bd1f69e295f0.png)
When I try to download it, only this package shows up:
![Screen Shot 2019-03-22 at 12 28 15 PM](https://user-images.githubusercontent.com/9019452/54838042-1d33f980-4c9e-11e9-991c-999802f2f8d9.png)

Reply:
I'm not a Sublime Text user so not really sure. 😕 Maybe @weerd might know!